foundationsync/indico_foundationsync.py

```diff
@@ -311,14 +313,21 @@ def fetch_room_equipment(self, connection, room_name=None):
         for room, equipment_types in foundation_room_equipment.iteritems():
             # We handle VC subequipment like equipment that's in the foundation DB since the latter only contains "VC"
             # but no information about the actually available vc equipment...
-            foundation_equipment_ids = set(equipment_types) | {eq.id for eq in vc_equipment}
+            vc_equipment_ids = {eq.id for eq in vc_equipment}
+            foundation_equipment_ids = set(equipment_types) | vc_equipment_ids
             for equipment in room.available_equipment.filter(~EquipmentType.id.in_(foundation_equipment_ids)):
-                self._logger.info("Mismatch: Room '%s' has equipment '%s' in Indico DB but not in Foundation",
-                                  room.full_name, equipment.name)
+                if equipment == vc_parent:
+                    for vc_equip in room.available_equipment.filter(EquipmentType.parent_id == vc_parent.id):
+                        room.available_equipment.remove(vc_equip)
+                        counter['deleted'] += 1
+                        self._logger.info("Deleted VC equipment '%s' from room '%s'", equipment.name, room.full_name)
```

Comment:
are you sure you want `equipment.name` here and not `', '.join(x.name for x in vc_equip)`?